space_age/spec/features/user_can_view_index_page_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature "user can view index page" do
+  scenario "user will see full options for images" do
+    destination = create_list(:destination, 3)
+    visit root_path
```

Comment:
The way we currently specced it out was that our root_path would be just the landing page. I implemented a welcome controller that just can pass that view. On that page, we agreed to have just the big image a button to the trips index. 

Are you shooting to have this test, test the landing page or the trips index? If it is the trip index, I believe we need to create trips, not destinations. If, this is for the landing page, are the destinations being created so we can see them on the side-nav? 

My understanding was that trips = items, destinations = categories. But I could have misunderstood! 

Thanks!